uni/analysis_options.yaml

```diff
@@ -1,38 +1,16 @@
-# This file configures the analyzer, which statically analyzes Dart code to
-# check for errors, warnings, and lints.
-#
-# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
-# IDEs (https://dart.dev/tools#ides-and-editors). The analyzer can also be
-# invoked from the command line by running `flutter analyze`.
+include: package:very_good_analysis/analysis_options.yaml
 
-# The following line activates a set of recommended lints for Flutter apps,
-# packages, and plugins designed to encourage good coding practices.
-include: package:flutter_lints/flutter.yaml
+analyzer:
+  # Exclude auto-generated files from dart analysis
+  exclude:
+    - '**.g.dart'
+    - '**.freezed.dart'
 
+# Custom linter rules. A list of all rules can be found at
+# https://dart-lang.github.io/linter/lints/options/options.html
 linter:
-  # The lint rules applied to this project can be customized in the
-  # section below to disable rules from the `package:flutter_lints/flutter.yaml`
-  # included above or to enable additional rules. A list of all available lints
-  # and their documentation is published at
-  # https://dart-lang.github.io/linter/lints/index.html.
-  #
-  # Instead of disabling a lint rule for the entire project in the
-  # section below, it can also be suppressed for a single line of code
-  # or a specific dart file by using the `// ignore: name_of_lint` and
-  # `// ignore_for_file: name_of_lint` syntax on the line or in the file
-  # producing the lint.
   rules:
-    # avoid_print: false  # Uncomment to disable the `avoid_print` rule
-    # prefer_single_quotes: true  # Uncomment to enable the `prefer_single_quotes` rule
-    always_use_package_imports: true
-    prefer_final_locals: true
-
-# Additional information about this file can be found at
-# https://dart.dev/guides/language/analysis-options
+    public_member_api_docs: false
+    avoid_equals_and_hash_code_on_mutable_classes: false
+    sort_pub_dependencies: false
```

Comment:
Any reason to disable this?

Reply:
`avoid_equals_and_hash_code_on_mutable_classes` makes the models to need a not so small refactor so i've ignored it for now, maybe it should be addressed on a later issue, because the rule makes sense but it's a bit out of scope for this pr imo.
`sort_pub_dependencies` is just a bit annoying and unnecessary imo.

Reply:
`public_member_api_docs` is just a hassle to document every function in the app, which is a bit unnecessary imo

Reply:
I think that `sort_pub_dependencies` is quite interesting since it reduces the time to look for a package in the list. `public_member_api_docs` is a future wish of mine, but I agree that it should be disabled for now. About `avoid_equals_and_hash_code_on_mutable_classes`, agree